modules/sdk/project-templates-workspace/src/main/resources/archetype-resources/gradle.properties

```diff
@@ -1,17 +1,113 @@
+    # downloading the bundle. The default value is false.
+    #
+    #liferay.workspace.bundle.token.download=false
+
+    #
+    # Set the email address to use when downloading a DXP bundle. This must
+    # match the email address registered for your DXP subscription. Only use
+    # this property if you do not want to use a workspace token to supply your
+    # user credentials (i.e., liferay.workspace.bundle.token.download=false).
+    #
+    #liferay.workspace.bundle.token.email.address=
+
+    #
+    # Set this to true to override the existing token with a newly generated
+    # token created by the "createToken" task. The default value is false.
+    #
+    #liferay.workspace.bundle.token.force=false
+
+    #
+    # Set the password to use when downloading a DXP bundle. This must match the
+    # password registered for your DXP subscription. Only use this property if
+    # you do not want to use a workspace token to supply your user credentials
+    # (i.e., liferay.workspace.bundle.token.download=false).
+    #
+    #liferay.workspace.bundle.token.password=
+
+    #
+    # Set the file to hold the Liferay bundle authentication token password.
+    # The default file value is "~/.liferay/token".
+    #
+    #liferay.workspace.bundle.token.password.file=
+
+    #
+    # Set the URL pointing to the bundle Zip to download. If the URL points to a
+    # DXP bundle (e.g., https://api.liferay.com/...), set the
+    # "liferay.workspace.bundle.token.download" property to true. The default
+    # value is the URL for Liferay Portal CE 7.0.3 GA4.
+    #
+    #liferay.workspace.bundle.url=https://cdn.lfrs.sl/releases.liferay.com/portal/7.0.3-ga4/liferay-ce-portal-tomcat-7.0-ga4-20170613175008905.zip
+
+    #
+    # Set this to true to configure Liferay CDN as the default repository in the
+    # root project. The default value is false.
+    #
+    #liferay.workspace.default.repository.enabled=false
+
+    #
+    # Set the environment with the settings appropriate for current development.
+    # The "configs" folder is used to hold different environments in the same
+    # workspace. You can organize environment settings and generate an
+    # environment installation with those settings. There are five
+    # environments: common, dev, local, prod, and uat. The default value is
+    # "local".
+    #
+    #liferay.workspace.environment=local
+
+    #
+    # Set the folder that contains the Liferay bundle downloaded from the
+    # "liferay.workspace.bundle.url" property. The default value is "bundles".
+    #
+    #liferay.workspace.home.dir=bundles
+
+    #
+    # Set this to true to configure Liferay CDN as the default repository for
+    # module/OSGi projects. The default value is false.
+    #
+    #liferay.workspace.modules.default.repository.enabled=false
+
+    #
+    # Set the folder that contains all module/OSGi projects. The default value
+    # is "modules".
+    #
+    #liferay.workspace.modules.dir=modules
+
```

Comment:
Should we also add `liferay.workspace.modules.jsp.precompile.enabled`? If `true` (default: `false`), the JSP files in the OSGi modules will be compiled and added to the dist zip/tar.